### PR TITLE
Stabilizing runtime detection tests

### PR DESCRIPTION
**Pull request by rhopp**

### CODE REVIEW

**Comment by psrna** [comment]
btw. cool branch name :)

File: tests/org.jboss.tools.runtime.as.ui.bot.test/src/org/jboss/tools/runtime/as/ui/bot/test/detector/cdk2/DetectCDK2.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package org.jboss.tools.runtime.as.ui.bot.test.detector.cdk2;
```
Comment (psrna) [inline tests/org.jboss.tools.runtime.as.ui.bot.test/src/org/jboss/tools/runtime/as/ui/bot/test/detector/cdk2/DetectCDK2.java]: Please add copyright info.

File: tests/org.jboss.tools.runtime.as.ui.bot.test/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -11,7 +11,6 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Require-Bundle: org.eclipse.ui,
  org.eclipse.core.runtime,
  org.eclipse.swtbot.swt.finder;bundle-version="2.0.0",
```
Comment (rawagner) [inline tests/org.jboss.tools.runtime.as.ui.bot.test/META-INF/MANIFEST.MF]: you can also remove this dep

**Comment by rhopp** [comment]
Both notes addressed. Whenever you are ready, @psrna.

**Pull request merged**